src/main/kotlin/no/nav/familie/ba/sak/kjerne/grunnlag/personopplysninger/bostedsadresse/GrBostedsadresse.kt

```diff
@@ -68,6 +68,14 @@ abstract class GrBostedsadresse(
         verdi = this.tilFrontendString(),
     )
 
+    fun tilKopiForNyPerson(nyPerson: Person): GrBostedsadresse =
+        tilKopiForNyPerson().also {
+            it.periode = periode
+            it.person = nyPerson
+        }
+
+    abstract fun tilKopiForNyPerson(): GrBostedsadresse
```

Comment:
For å unngå at man risikerer å kalle denne uten person
```suggestion
    protected abstract fun tilKopiForNyPerson(): GrBostedsadresse
```